meta-mender-core/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -1,4 +1,4 @@
-PACKAGECONFIG += "networkd resolved"
+PACKAGECONFIG_append = "networkd resolved"
```

Comment:
I'm trying to find a reason why we even enable `networkd` & `resolved` in meta-mender-core. Aside from shipping networkd config files, which should really be a part of meta-mender-demo, there's nothing that ties us to networkd. Similarly, `resolved` is enabled for the build, but unless you add `resolve` to `nsswitch.conf` it's not going to be used. I've posted a patch that adds resolve to nsswitch http://lists.openembedded.org/pipermail/openembedded-core/2016-September/126738.html but didn't bother to follow up after systemd got updated. Also, IIRC you won't be even able to use `resolved` if you build with musl (as there was no nsswitch support). 

Lastly, there's no real dependency on systemd either.